plugins/data-types/src/index.ts

```diff
@@ -149,7 +165,9 @@ export class DataTypes {
 			seriesData.values = convertToString(seriesData.values)
 			seriesData.type = 'dimension'
 
-			unitCheck(seriesData)
+			if (this.dataTypesOptions.units !== false) {
```

Comment:
Why the double negation? maybe `=== true`
Why not  `if (this.dataTypesOptions.units) {`